app/controllers/application_controller.rb

```diff
@@ -45,7 +45,9 @@ def cors_set_headers
   end
 
   def current_season
-    @season ||= Season.current
+    #TEMP solution to find the application form
+    #@season ||= Season.current
+    @season = Season.find(2)
```

Comment:
What is the reason for this switch? There's no guarantee that in every environment the ID of the current season is "2".